src/app.ts

```diff
@@ -514,6 +573,7 @@ function registerAuthenticationRoutes(app: Express, basePath: string): void {
     toExpressHandler(acsController.slo, acsController)
   );
 
+  // TODO: This route need to be reloaded after clearAndReloadSpidStrategy?
```

Comment:
@cloudify maybe can be an issue with `/metadata` API call. When the `SpidStrategy` object is regenerated, if changes are present in it, currently the `/metadata` API call i think that it will returns not updated information. is my hypothesis real?

Reply:
good point, afaik changes in `SpidStrategy` will only be related to IdPs, this service `/metadata` shouldn't change at all - isn't `SpidStrategy` only relevant for passport authentication?